modules/cnv-virtctl-commands.adoc

```diff
@@ -15,27 +15,34 @@ the {CNVProductName} documentation.
 |===
 |Command |Description
 
-|`virtctl start <vm>` 
+|`virtctl start <vm_name>`
 |Start a virtual machine.
 
-|`virtctl stop <vm>` 
+|`virtctl stop <vm_name>`
 |Stop a virtual machine.
 
-|`virtctl restart <vm>` 
+|`virtctl pause vm\|vmi <object_name>`
+| Pause a virtual machine or virtual machine instance. The machine state is kept
+in memory.
+
+|`virtctl unpause vm\|vmi <object_name>`
+| Unpause a virtual machine or virtual machine instance.
+
+|`virtctl restart <vm_name>`
 |Restart a virtual machine.
 
-|`virtctl expose <vm>` 
+|`virtctl expose <vm_name>`
 |Create a service that forwards a designated port
 of a virtual machine or virtual machine instance and expose the service on
 the specified port of the node.
 
-|`virtctl console <vmi>` 
+|`virtctl console <vmi_name>`
 |Connect to a serial console of a virtual machine instance.
 
-|`virtctl vnc <vmi>` 
+|`virtctl vnc <vmi_name>`
 |Open a VNC connection to a virtual machine instance.
 
-|`virtctl image-upload <...>` 
+|`virtctl image-upload <...>`
```

Comment:
Gosh, this must be old. Can you please update the `<...>` to something like '<image-name>'